client/packages/common/src/utils/formatters/JSXFormatters.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export const JSXFormatters = {
+  /*
+  Converts a string with line breaks (\n) into HTML with <br/> line breaks
+  */
+  replaceHTMLlineBreaks: (input: string) => {
+    const lines = input.split('\n');
+    return lines.map((line, i) => (
+      <span key={i}>
+        {line}
+        {/* Don't add line break to final line */}
+        {i === lines.length - 1 ? null : <br />}
+      </span>
+    ));
+  },
+};
```

Comment:
This seemed like a useful utility that might come in handy elsewhere, so I've put it in a new utils file. Not sure of the best place for this, but didn't quite seem to fit in with the other utils. Any particular suggestions, @mark-prins ?

Reply:
Not required - just add `whitespace` when displaying, like so:
`<Typography style={{ whiteSpace: 'pre' }}>`

Other comments.. the use of `<br/>` tags as layout controls is frowned upon, though I'd use them in this example! Ideally each line should be wrapped in `Typography` 🙄

Reply:
Cool, that works thanks. The only thing we lose is I added a little bit of additional functionality (in the next PR #1230) that also strips out blank lines, which is useful if a field in the displayString is not present. Minor thing, but looks a little nicer. I'll have a think about another way to achieve the same thing when I revisit that PR shortly.

Reply:
I've added a "removeEmptyLines" regex utility in the subsequent PR to achieve this.